os/drivers/input/ist415.c

```diff
@@ -340,6 +351,9 @@ static int ist415_process_event(struct ist415_dev_s *dev)
 			// TODO: Report PalmLarge Off Process
 			ist415vdbg("PalmLarge Off (T)\n");
 		}
+#if defined(CONFIG_IST415_SAMPLING_RATE)
+		ist415_measure_sample_rate(dev, dev->touch_pressed_num);
```

Comment:
I think it will give build warning for type mismatch.
Let's add typecasting here
```
(bool)dev->touch_pressed_num
```